build-pine64-image.sh

```diff
@@ -23,64 +66,150 @@ if [ "$(id -u)" -ne "0" ]; then
 	exit 1
 fi
 
-if [ -z "$DISTRO" ]; then
-	DISTRO="xenial"
+echo -n "Checking for presence of zerofree... "
+
+if [ -x /usr/sbin/zerofree ]; then
+	echo "OK"
+else
+	echo "Not Installed"
+	echo "ERROR: You need to install zerofree in order to generate a pine64 image."
```

Comment:
Please do not introduce additional dependencies. If you insist on having this, make it optional and create a seperate pull request.